velox/exec/MergeJoin.cpp

```diff
@@ -271,9 +271,27 @@ void MergeJoin::addOutputRowForLeftJoin(
     target->setNull(outputSize_, true);
   }
 
```

Comment:
for safety, should we have a check that this function is only called in left joins (and the same for the one below)?

Reply:
@pedroerp Updated.